services/streammanager/kafka/kafkamanager.go

```diff
@@ -1,25 +1,27 @@
 package kafka
 
 import (
-	"crypto/sha256"
-	"crypto/sha512"
-	"crypto/tls"
-	"crypto/x509"
+	"context"
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"strconv"
 	"strings"
 	"time"
 
-	"github.com/Shopify/sarama"
 	"github.com/tidwall/gjson"
-	"github.com/xdg/scram"
 
 	"github.com/rudderlabs/rudder-server/config"
-	"github.com/rudderlabs/rudder-server/utils/logger"
+	"github.com/rudderlabs/rudder-server/services/streammanager/kafka/client"
+	rslogger "github.com/rudderlabs/rudder-server/utils/logger"
```

Comment:
Any specific reason we had to name it as rslogger?

Reply:
Clash with local interface.